app/models/settings/campaign.rb

```diff
@@ -5,6 +5,7 @@ class Campaign < Base
     # Define your settings
     setting :articles_expiry_time, type: :integer, default: 4
     setting :articles_require_approval, type: :boolean, default: 0
+    setting :display_name, type: :string, default: ""
```

Comment:
nit-pick, non-blocking: these look to have been alphabetized before, maybe this should be one line farther down?